keras_nlp/tests/test_case.py

```diff
@@ -417,3 +418,6 @@ def compare(actual, expected):
                 self.assertAllClose(actual, expected, atol=0.01, rtol=0.01)
 
             tree.map_structure(compare, output, expected_partial_output)
+
+    def get_test_data_dir(self):
+        return pathlib.Path(__file__).parent / "test_data"
```

Comment:
Nit, but I think returning a simple string might be a little less surprising to people here.

Ok to use pathlib, but cast to string before return, and then use `os.path.join(self.get_test_data_dir(), "file")` from the tests.

The will match our [usage of get_temp_dir](https://github.com/search?q=repo%3Akeras-team%2Fkeras-nlp%20get_temp_dir&type=code).

Reply:
Ah, got it!